.travis.yml

```diff
@@ -31,6 +33,7 @@ script:
   - "! git grep -I ' $' ':(exclude)Firestore/Protos/nanopb'"
   - "! git grep -EL --name-only 'Copyright [0-9]{4}.*Google' | grep -v third_party | egrep '\\.(m|h|cc|mm|c)$'"
   - ./scripts/style.sh test-only # Validate clang-format compliance
+  - ./scripts/lint.sh # Validate Google style compliance
```

Comment:
This is still failing in the latest commit.

Update the comment to "Google C++ style compliance"

If it shows on travis as taking more than a few seconds, please move into Firestore/test.sh

Reply:
Updated.

Sure, I'll keep an eye on the run time. Right now it runs on Travis for 5 sec, 6 sec for the two runs (the time may be different on after a fixed commit). I'll wait for the head fixed before merge this PR.

Reply:
It's probably not a bad idea to move this into Firestore/test.sh anyway since that makes it easier to check Firestore in one swell foop.

Reply:
I'm not sure moving a style-check into a test script is not confusing. Let me move the check below right above the test.sh. So the output will be together. But make them separate. And let it conditions on Firestore change only.

Updated. PTAL.